data-products/src/main/scala/org/sunbird/analytics/job/report/StateAdminReportJob.scala

```diff
@@ -424,7 +424,7 @@ object StateAdminReportJob extends optional.Application with IJob with StateAdmi
     val locationIdList = udf[List[String], String](locationIdListFunction)
     
     def profileTypeFunction(profileType: String) : String = {
-        if(profileType != null) {
+        if(profileType != null && profileType != "") {
```

Comment:
Convert it to Option variable and check for emptiness

Reply:
@SanthoshVasabhaktula changes done.